packages/youtube/commands/youtube/backsync_comments.ts

```diff
@@ -39,13 +43,13 @@ export async function handler() {
       const sourceId = comment.get('sourceId') as string;
       const status = decision.get('status');
 
-      if (status === 'Defer') {
+      if (status === MODERATION_ACTION_DEFER) {
         logger.info('Not syncing comment %s:%s - in deferred state', comment.id, sourceId);
         markDecisionExecuted(decision);
         return;
       }
 
-      const moderationStatus = (status === 'Accept') ? 'published' : 'rejected';
+      const moderationStatus = (status === MODERATION_ACTION_ACCEPT) ? 'published' : 'rejected';
```

Comment:
maybe for later (make a TODO comment perhaps), but 'published' and 'rejected' probably want to be constants.

Reply:
These values are defined by the youtube API.  It's a shame they don't define constants themselves, or at least make the type strict.

Reply:
nice.